src/js/components/ContentInfo/index.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react';
+import FilmItem from '../FilmItem';
+import type { Film } from '../../flow-types.js';
+
+type Props = {
+ onFilmSelect: (id: number) => void,
+ films: Array<Film>
+}
+
+const ContentInfo = (props: Props) => (
+  <div className='content-info'>
+    <ul className='film-list'>
+      {props.films.length && props.films.length ? (
```

Comment:
This condition doesn't really make sense 😄 
Maybe just: `props.films.length ? ...`